src/main/java/frc/trigon/robot/utilities/mechanisms/ElevatorMechanism2d.java

```diff
@@ -0,0 +1,63 @@
+package frc.trigon.robot.utilities.mechanisms;
+
+import edu.wpi.first.wpilibj.smartdashboard.Mechanism2d;
+import edu.wpi.first.wpilibj.smartdashboard.MechanismLigament2d;
+import edu.wpi.first.wpilibj.smartdashboard.MechanismRoot2d;
+import edu.wpi.first.wpilibj.util.Color8Bit;
+import org.littletonrobotics.junction.Logger;
+
+/**
+ * A Mechanism2d object to display the current position and target position of an elevator.
+ */
+public class ElevatorMechanism2d {
+    private static final double MECHANISM_STARTING_ANGLE = 90;
+    private final String key;
+    private final Mechanism2d mechanism;
+    private final MechanismLigament2d
+            currentPositionLigament,
+            targetPositionLigament;
+
+    public ElevatorMechanism2d(String key, double maximumDisplayablePosition) {
+        this(key, maximumDisplayablePosition, MechanismConstants.BLUE);
+    }
+
+    public ElevatorMechanism2d(String key, double maximumDisplayablePosition, Color8Bit mechanismColor) {
```

Comment:
have params of `minimumLength` and `maximumLength`, then make sure each input gets added the minimum length, and that the maximum length is the hight of the mechanism. 
document this.